libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/GuidanceViewImageProvider.kt

```diff
@@ -83,17 +65,19 @@ class GuidanceViewImageProvider() {
 
     /**
      * The API allows you to cancel the rendering of guidance view.
-     * @param context Context
      */
-    fun cancelRender(context: Context) {
-        if (targets.isNotEmpty()) {
-            for(target in targets) {
-                Picasso.Builder(context).build().cancelRequest(target)
-            }
-            targets.clear()
-        }
+    fun cancelRender() {
+        ioJobController.job.cancelChildren()
+        mainJobController.job.cancelChildren()
     }
 
+    private suspend fun getBitmapAsync(url: String): Bitmap =
+        withContext(ioJobController.scope.coroutineContext) {
```

Comment:
```suggestion
        withContext(ThreadController.IODispatcher) {
```
and remove
`private val ioJobController: JobControl = ThreadController.getIOScopeAndRootJob()`